app/representers/backend_api_representer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module BackendApiRepresenter
+  include ThreeScale::JSONRepresenter
+
+  wraps_resource
+
+  property :id
+  property :name
+  property :system_name
+  property :description
+  property :private_endpoint
+  property :account_id
+  property :created_at
+  property :updated_at
```

Comment:
We should also add hypermedia to metrics and proxy_rules

Reply:
I can't yet because those routes do not exist, but I added the comments and I will reference the comments from those jira issues as well.